index.html

```diff
@@ -1,21 +1,106 @@
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" href="css/main.css">
 
     <title>Responsive grid project</title>
 </head>
 
 <body>
+    <div class="contener">
+        <div class="text_area_nav">
+        <div class="text_nav1">
+            <div class="scouer"></div>
+
+            <h1 class="title_nav">Maya Nelson</h1>
+        </div>
+        <div class="text_nav2">
+            <h2>PROJECT MANAGER</h2>
+        </div>
+   
+    <div class="menu_bar">
+        <input id="toggle" type="checkbox"></input>
+        <label for="toggle" class="hamburger">
+            <div class="top-bun"></div>
+            <div class="meat"></div>
+            <div class="bottom-bun"></div>
+        </label>
+
+        <div class="nav">
+            <div class="nav-wrapper">
+                <nav>
+                    <a href="./index.html">ABOUT ME</a><br>
+                    <a href="./resume.html">RESUME</a><br>
+                    <a href="./projects.html">PROJECT</a><br>
+                    <a href="./contact.html">CONTACT</a>
+                </nav>
+            </div>
+        </div>
+    </div>
+</div>
+        <main>
+            <section class="info_maya">
+                <img src="img/me.png" alt="background_img" >
+                <h1 class="h1_1">Maya
+                    Nelson</h1>
+                <h1 class="underlin">______</h1>
+                <br>
+                <div class="boutton_main">
+                    <a href="./resume.html"><button>RESUME</button></a>
+                    <a href="./projects.html"><button>PROJECTS</button></a>
+                </div>
+                <div class="icon_media">
+                    <img src="img/facbook.png" alt="facebook-icon" class="icon">
+                    <img src="img/twetter.png" alt="twitter-icon" class="icon">
+                    <img src="img/linkin.webp" alt="linkin_icon" class="icon">
+                    <img src="img/instagram-logo-A807AD378B-seeklogo.com.png" alt="instgram" class="icon">
+                </div>
+            </section>
+            <section class="content_text" >
+                <h1 class="h1_2">Hello</h1>
+                <h2 class="h2_1">Here's who I am & what I do </h2>
+                <p>I'm a paragraph. Click here to add your own text and edit me. It’s easy. Just click “Edit Text” or
+                    double click me to add your own content and make changes to the font.
+
+                </p>
+                <p>I’m a great place for you to tell a story and let your users know a little more about you.
 
-    <!-- Add your markup here -->
+                </p>
 
+            </section>
+        </main>
+        <footer>
+            <div class="footer_1">
+                <div class="call_us">
+                    <h1  class="h2_1">Call</h1>
```

Comment:
You should never use h1 in footer at all, that is because the footer content meant to be quite small, also like I said only use h1 one time in the page if you can